scripts/bootstrap/bootstrap.sh

```diff
@@ -29,41 +29,29 @@ if [ -n "${EMBED_LABEL}" ]; then
     EMBED_LABEL_ARG=(--stamp --embed_label "${EMBED_LABEL}")
 fi
 
-: ${JAVA_VERSION:="1.8"}
-
-if [ "${JAVA_VERSION}" = "1.7" ]; then
-  _BAZEL_ARGS="--java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain_jdk7 \
-        --host_java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain_jdk7 \
-        --spawn_strategy=standalone \
-        --nojava_header_compilation \
-        --define JAVA_VERSION=1.7 --ignore_unsupported_sandboxing \
-        --compilation_mode=opt \
-        ${EXTRA_BAZEL_ARGS:-}"
-else
-  _BAZEL_ARGS="--java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain \
-        --host_java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain \
-        --spawn_strategy=standalone \
-        --nojava_header_compilation \
-        --strategy=Javac=worker --worker_quit_after_build --ignore_unsupported_sandboxing \
-        --compilation_mode=opt \
-        ${EXTRA_BAZEL_ARGS:-}"
-fi
+_BAZEL_ARGS="--java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain \
+      --host_java_toolchain=//src/java_tools/buildjar:bootstrap_toolchain \
+      --spawn_strategy=standalone \
+      --nojava_header_compilation \
+      --strategy=Javac=worker --worker_quit_after_build --ignore_unsupported_sandboxing \
+      --compilation_mode=opt \
+      ${EXTRA_BAZEL_ARGS:-}"
 
 if [ -z "${BAZEL-}" ]; then
   function _run_bootstrapping_bazel() {
     local command=$1
     shift
     run_bazel_jar $command \
         ${_BAZEL_ARGS} --verbose_failures \
-        --javacopt="-g -source ${JAVA_VERSION} -target ${JAVA_VERSION}" "${@}"
```

Comment:
It might be better to keep the -source -target specification for machines that have an even newer JDK installed (or older, in case we don't check the version somewhere?).

Reply:
Also, you might want to keep JAVA_VERSION as a constant in a central location for the next JDK upgrade.